src/interface_adapter/create_recipe/CreateRecipeController.java

```diff
@@ -1,4 +0,0 @@
-package interface_adapter.create_recipe;
-
-public class CreateRecipeController {
```

Comment:
why is it deleted?

Reply:
Because we already have upload recipe, so we do not need the create recipe use case, which represent the same thing.